### PR TITLE
[JENKINS-21327] replace RUN_SCRIPTS with a new Scriptler RUN permission

### PR DESCRIPTION
**Pull request by imod**
this should replace the use of `RUN_SCRIPT` with a new Scriptler specific `RUN` permisson

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/scriptler/Messages.properties [hunk 1]
```diff
@@ -39,4 +39,6 @@ skipParamter = skipping parameter [{0}] this name is used internal, please renam
 resultPrefix = Result:  
 no_parameters_defined = There is not any parameter defined for this job.
 tokenmacro_AdminScriptOnly = The script [{0}] exists, but is marked to be used by admins only and is therefore not allowed for usage in the TokenMacro.
-tokenmacro_ScriptDoesNotExist = No script with the id [{0}] could be found.
\ No newline at end of file
+tokenmacro_ScriptDoesNotExist = No script with the id [{0}] could be found.
+permissons_title = Scriptler
+permissons_run_description = allow execution of scritpler scripts
```
Comment (jglick) [inline src/main/resources/org/jenkinsci/plugins/scriptler/Messages.properties]: typo

File: src/main/java/org/jenkinsci/plugins/scriptler/ScriptlerManagement.java [hunk 2]
```diff
@@ -65,6 +68,9 @@
     private final static String ALL = "(all)";
     private final static String ALL_SLAVES = "(all slaves)";
 
+    public static final PermissionGroup SCRITPLER_PERMISSONS = new PermissionGroup(ScriptlerManagement.class, Messages._permissons_title());
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/scriptler/ScriptlerManagement.java]: typo

**Comment by imod** [comment]
@jglick hmm, from my POV this is logical consequence if I want to allow some users (non admins) to execute/use scriptler scripts. `RUN_SCRIPT` obviously was wrong, but I don't quite understand your confusion with the new independent permission, can you explain?

**Comment by imod** [comment]
@daniel-beck do you understand what @jglick is talking about? I really don't get it anymore some more context information in the answers would really help...

**Comment by daniel-beck** [comment]
Probably best to try to make scripts `AccessControlled` and implement properly fine-grained permissions control if this is the route you want to go.

I hope I'll have the time in the next two weeks to investigate this further.

**Comment by daniel-beck** [comment]
Superseded by #32, I think.

**Comment by imod** [comment]
closing because of #32

**Pull request closed without merging**